src/handlebars/support.handlebars

```diff
@@ -59,6 +59,360 @@
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+				</tr>
+				<tr>
+					<td>Windows Server 2012</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+				</tr>
+				<tr>
+					<td>Windows Server 2012 R2</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+				</tr>
+				<tr>
+					<td>Windows Server 2016</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+				</tr>
+			</tbody>
+		</table>
+
+		</br>
+
+		<table border="1">
+			<thead>
+				<tr>
+					<th>macOS</th>
+					<th>x64</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>OSX 10.10</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span></td>
+				</tr>
+			</tbody>
+		</table>
+
+		</br>
+
+		<table border="1">
+			<thead>
+				<tr>
+					<th>AIX</th>
+					<th>ppc64</th>
+				</tr>
+			</thead>
+			<tbody>
+				<tr>
+					<td>AIX 7.1 TL4</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+				</tr>
+				<tr>
+					<td>AIX 7.2</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+				</tr>
+			</tbody>
+		</table>
+
+		<p>When public support for an operating system version ends, OpenJDK builds can no longer be supported on that level.</p>
```

Comment:
I'm not 100% of this statement, I think it should be removed.

Reply:
Removed